### PR TITLE
SUMMARY.md file for Gitbook

### PR DESCRIPTION
**Pull request by kvetoslavnovak**
I have added SUMMARY.md file so you can merge it because of Gitbook.

### CODE REVIEW

**Comment by tmeasday** [comment]
@kvetoslavnovak can you explain what we could do if we merged this?

**Comment by kvetoslavnovak** [comment]
Automaticaly generated GitBook (pdf,...).
See my discussion and walkthrough with stubailo here #166

**Comment by stubailo** [comment]
More context: https://forums.meteor.com/t/meteor-guide-in-pdf-mobi-epub-gitbook/28872/3

**Comment by abernix** [comment]
@kvetoslavnovak Is there anything we can still do to save this PR?  I think there's just one change that I would object to right now if this does make it possible to implement the GitBook format.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Meteor Guide
 
-[Read the guide!](http://guide.meteor.com/)
+The Gitbook version of the Meteor Guide [http://guide.meteor.com/](http://guide.meteor.com/)
```
Comment (abernix) [inline README.md]: I don't think we would want to change this.

**Comment by abernix** [comment]
With no familiarity with this, and as the original author is no longer working on this, I think we'll close this.

**Pull request closed without merging**